src/stores/SchemaStore.ts

```diff
@@ -108,6 +112,10 @@ export class SchemaStore {
 		this.selectedSchema = schema;
 	};
 
+	setSelectedDictionary = (dictionary: DictionaryEntity | null) => {
```

Comment:
this should be an action, also i think it's better to rename `setSelectedDictionary` to selectDictionary, i've already renamed `setSelectedSchema` to `selectSchema` in my pr